yongkeun/week2/과제/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>2주차 과제</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <section class="mainbox">
```

Comment:
```suggestion
    <section class="main-box">
```
html, css에서는 단어의 연결을 "-"을 사용해 표현합니다.
mainbox보다는 main-box가 더 좋아요 ㅎㅎ